prow/plugins/branchcleaner/README.md

```diff
@@ -0,0 +1,14 @@
+# Branchcleaner
+
+The `branchcleaner` makes Prow automatically delete the source branch of a Pull Request if it is
+on the same repo as the target branch. This is helpful to keep repos that don't allow forking clean.
```

Comment:
I think it would be good to mention that the branch is only deleted if the PR is merged.

Reply:
Oups, very good point